src/plugins/JavaBIPEngine/JavaBIPEngine.js

```diff
@@ -124,31 +159,66 @@ define([
                                     }
                                 }
                                 fs.writeFileSync(path + '/' + fileName, filesToAdd[fileName], 'utf8');
-                                // child = exec('mvn test -Dtest=' + testInfo.className, function (error, stdout, stderr) {
-                                //         console.log('stdout: ' + stdout);
-                                //         console.log('stderr: ' + stderr);
-                                //         filesToAdd[engineOutputFileName] = stdout;
-                                //         if (error !== null) {
-                                //             throw new Error('Execution error: ' + error + '.');
-                                //         }
-                                //     });
-                                // fs.writeFileSync(path + '/' + engineOutputFileName, filesToAdd[engineOutputFileName], 'utf8');
-                            }
+                                for (file in behaviorFiles) {
+                                    compileCode += 'javac -cp "' + process.cwd() + '/engineLibraries/*" ' + path + '/' + file + '\n\n';
+                                }
+                                compileCode += 'javac -cp "' + path + '/:' + process.cwd() + '/engineLibraries/*" ' + path + '/' + fileName;
 
+                                fs.writeFileSync(path + '/compile.sh', compileCode, 'utf8');
+                                self.sendNotification('Compilation script has been successfully created.');
```

Comment:
I would change this to:
`Compiling java code..`

`Compilation successful.`

`Calling simulate..`

`Simulate successful.`

Or something like that.